app/views/planning_applications/validate_form.html.erb

```diff
@@ -66,8 +66,8 @@
           </h2>
           <div class="govuk-error-summary__body">
             <ul class="govuk-list govuk-error-summary__list">
-              <% @planning_application.errors.full_messages.each do |error| %>
```

Comment:
Why aren't we using the full messages anymore?

Reply:
Because when using the full messages, the message displayed becomes
"Planning application lalalal error"
since the errors are all being shoved into :planning_application.

This is actually what shows up in staging right now when you remove the date, but no one seems to have noticed the bug:
![image](https://user-images.githubusercontent.com/9452321/133234869-d0f2c9d5-5116-4070-8789-b4a4aca83cdd.png)

I have the idea I changed this somewhere else before but some reason we went back to the full_messages. When I upgraded the app to rails 6, the error messages changed in that version of rails and hence needing reworking as the errors are no longer a hash - now each error knows about itself: https://code.lulalala.com/2020/0531-1013.html

Reply:
Gotcha